--- standalone.conf.final	2015-09-17 15:27:52.060440200 -0300
+++ standalone.conf	2015-09-17 17:01:09.067956000 -0300
@@ -50,12 +50,6 @@
    JAVA_OPTS="-Xms1303m -Xmx1303m -XX:MaxPermSize=256m -Djava.net.preferIPv4Stack=true"
    JAVA_OPTS="$JAVA_OPTS -Djboss.modules.system.pkgs=$JBOSS_MODULES_SYSTEM_PKGS -Djava.awt.headless=true"
    JAVA_OPTS="$JAVA_OPTS -Djboss.modules.policy-permissions=true"
-   if [ "$KEYCLOAK_IMPORT" ]; then
-      if $cygwin; then
-         KEYCLOAK_IMPORT=`cygpath -m "$KEYCLOAK_IMPORT"`
-      fi
-      JAVA_OPTS="$JAVA_OPTS -Dkeycloak.import=\"$KEYCLOAK_IMPORT\""
-   fi
    JAVA_OPTS="$JAVA_OPTS -Duser.language=en"
    JAVA_OPTS="$JAVA_OPTS -Dfile.encoding=UTF-8"
 else
