--- standalone.conf.original	2015-08-07 11:59:54.962743174 -0300
+++ standalone.conf	2015-08-07 13:39:01.419998568 -0300
@@ -50,6 +50,9 @@
    JAVA_OPTS="-Xms1303m -Xmx1303m -XX:MaxPermSize=256m -Djava.net.preferIPv4Stack=true"
    JAVA_OPTS="$JAVA_OPTS -Djboss.modules.system.pkgs=$JBOSS_MODULES_SYSTEM_PKGS -Djava.awt.headless=true"
    JAVA_OPTS="$JAVA_OPTS -Djboss.modules.policy-permissions=true"
+   JAVA_OPTS="$JAVA_OPTS -Dkeycloak.import=\"$JBOSS_HOME/myapp-realm.json\""
+   JAVA_OPTS="$JAVA_OPTS -Duser.language=en"
+   JAVA_OPTS="$JAVA_OPTS -Dfile.encoding=UTF-8"
 else
    echo "JAVA_OPTS already set in environment; overriding default settings with values: $JAVA_OPTS"
 fi
